geode-docs/developing/data_serialization/PDX_Serialization_Features.html.md.erb

```diff
@@ -31,6 +31,33 @@ When you serialize an object using PDX, <%=vars.product_name%> stores the object
 
 This centralization of object type information is advantageous for client/server installations in which clients and servers are written in different languages. Clients pass registry information to servers automatically when they store a PDX serialized object. Clients can run queries and functions against the data in the servers without compatibility between server and the stored objects. One client can store data on the server to be retrieved by another client, with no requirements on the part of the server.
 
+**Note:**
+There are situations where some of the information in the central registry of the PDX domain object metadata is lost, e.g. when restoring an old backup with an outdated central registry.
+
+When that happens, new clients connecting to the cluster for the first time will get outdated PDX type information from the central registry, but, since information in the central registry is cached by clients, old clients may have fresher information about PDX types than the central registry does. That will result into inconsistent information about PDX types spread across the system:
+
+-   old clients have fresh information
+-   the central registry has outdated information
+-   new clients have outdated information
+
+If old clients write entries of a PDX type they know but the central registry doesn't, new clients will get "Unknown PDX type" errors when they read those objects.
+
+To avoid this problem, clients may be configured with the system property in the table below to clear their PDX type cache when they disconnect from the cluster. After clearing their cache, old clients will re-generate type information for all PDX types, including the types the central registry "forgot". Since new PDX type information will be written in the central registry before entries of that type are written in the cluster, the central registry and all clients, old and new, will store consistent PDX type information.
+
+| Name                                      | Description                                             | Default |
+|-------------------------------------------|---------------------------------------------------------|---------|
+| `gemfire.ON_DISCONNECT_CLEAR_PDXTYPEIDS`  | Clear pdxType ids when client disconnects from servers. | `false` |
+-   In the API, set the System properties before the cache creation call. Example:
+
+    ``` pre
+    System.setProperty(PoolImpl.ON_DISCONNECT_CLEAR_PDXTYPEIDS, "true");
+    ```
+-   At the java command line, pass in System properties using the -D switch. Example:
+
+    ``` pre
+    java -Dgemfire.ON_DISCONNECT_CLEAR_PDXTYPEIDS=true
+    ```
+
```

Comment:
I would not include here how to set the variable in the different clients. Otherwise, you would also have to add the way to set the property in the native client.

Reply:
It would probably be better to add this variable and description to the corresponding documents of the Java and Native clients:
https://geode.apache.org/docs/geode-native/cpp/115/configuring/sysprops.html
https://geode.apache.org/docs/guide/115/reference/topics/gemfire_properties.html